docker/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  mapfish-print-manager:
```

Comment:
It would be nice to have a docker configuration for V2 as well. Do you see any chance to implement this? If not, just ignore.

Reply:
For the moment I'd ignore it to get progress on some other project. But I've added an issue as reminder --> https://github.com/terrestris/mapfish-print-manager/issues/154